### DISCUSSION TITLE
Setting RX an TX Pin in GUI...

### DISCUSSION THREAD

**Discussion by fisch55** [dzungpv/mitsubishi2MQTT#38] [General]
Hi, first of all THX for your Project and your work in it.
Got a Qusetion... Using ESP8266-ESP01 ---looks like the D1 min... Do I have to set manually any Pins or not. Got the same ESP like on the pictures on the site.
THX for reply

<img width="334" alt="Bildschirmfoto 2024-08-11 um 08 55 55" src="https://github.com/user-attachments/assets/22c71894-69d5-4df6-9388-624bc9a19026">

**Comment by dzungpv** [comment]
Set TX and RX pin only available if you are using ESP32
